int32 lpId
int32 unused
float64 timeOnTraj
float64 time
float64 ic_a
float64 ic_v
float64 ic_x
float64 jerk

